dao/src/test/java/greencity/repository/AdviceTranslationRepoTest.java

```diff
@@ -23,6 +30,35 @@
     @Autowired
     private AdviceTranslationRepo adviceTranslationRepo;
 
+    private Advice advice = Advice.builder().id(1L)
+            .habit(Habit.builder().id(1L).image("image_one.png")
+                    .build())
+            .build();
+
+    private AdviceTranslation firstAdviceTranslation = AdviceTranslation.builder()
+            .id(1L)
+            .language(Language.builder().id(1L).code("uk").build())
+            .content("Привіт")
+            .advice(advice).build();
+
+    private AdviceTranslation secondAdviceTranslation = AdviceTranslation.builder()
+            .id(2L)
+            .language(Language.builder().id(2L).code("en").build())
+            .content("Hello")
+            .advice(Advice.builder().id(2L)
+                    .habit(Habit.builder().id(2L).image("image_two.png")
+                    .build())
+            .build()).build();
+
+    private AdviceTranslation thirdAdviceTranslation = AdviceTranslation.builder()
+            .id(3L)
+            .language(Language.builder().id(3L).code("ru").build())
+            .content("Привет")
+            .advice(Advice.builder().id(3L)
+                    .habit(Habit.builder().id(3L).image("image_three.png")
+                            .build())
+                    .build()).build();
+
```

Comment:
Please move these objects creation to ModelUtils class and use them as local variables.

Reply:
Moved to ModelUtils.